mmdeploy/codebase/mmdet/models/utils/misc.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmdet.models.utils.misc.generate_coordinate')
+def generate_coordinate__onnx(ctx, featmap_sizes, device='cuda'):
```

Comment:
There is no `generate_coordinate` in `mmdet.models.utils.misc`.
Has this function be used?